platform/ios/src/UIImage+MGLAdditions.mm

```diff
@@ -39,10 +53,30 @@ - (nullable instancetype)initWithMGLPremultipliedImage:(const mbgl::Premultiplie
 }
 
 - (std::unique_ptr<mbgl::style::Image>)mgl_styleImageWithIdentifier:(NSString *)identifier {
+    mbgl::style::ImageStretches stretchX = {{
+        self.capInsets.left / self.scale, (self.size.width - self.capInsets.right) / self.scale,
+    }};
+    mbgl::style::ImageStretches stretchY = {{
+        self.capInsets.top / self.scale, (self.size.height - self.capInsets.bottom) / self.scale,
+    }};
+    
+    mbgl::optional<mbgl::style::ImageContent> imageContent;
+    if (!MGLEdgeInsetsIsZero(self.capInsets)) {
+        imageContent = (mbgl::style::ImageContent){
+            .left = static_cast<float>(self.capInsets.left * self.scale),
```

Comment:
I'm confused by the `* self.scale` here, compared with the `/ self.scale` above. What's going on here? (The macOS version below has `*` in both)

Reply:
`-initWithMGLStyleImage:` divides by the scale and `-mgl_styleImageWithIdentifier:` multiplies by the scale on both platforms.

Reply:
Oh, I see what you mean – you’re referring to the `stretchX` and `stretchY` variables. Yes, I think the division above is incorrect. The tests aren’t catching the mistake because they only assert on the image’s content rect, not its stretchable areas per se.